ruler/test_ruler.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import unittest
+import ruler
+from regular_shapes import disc, rounded_square
+
+
+class TestRuler(unittest.TestCase):
+    def setUp(self):
```

Comment:
There is no logic or parameterization being done in this `setUp()` routine. Therefore, it is better to set these constants at the module level, i.e.
```python
RESOLUTION = 1
PHYS_SIZE = (200, 200)
```
above the class definition.